packages/web-styles/.gitignore

```diff
@@ -3,7 +3,6 @@ node_modules/
 out-tsc/
 debug.log
 npm-debug.log
-src/**/*.js
```

Comment:
if .js files are now allowed in the entire /web-styles/src directory, the exceptions below become obsolete
- [ ] check if statments below (beginning with `!src/...` become obsolete